src/renderer/components/layout/setting-layout.scss

```diff
@@ -228,7 +228,7 @@
       margin-top: 0;
       margin-bottom: 8px;
       padding-bottom: 0;
-      font-size: 12px;
```

Comment:
Why this change?

Reply:
For better readability. I found this font too small to read comfortably.

Reply:
Sounds good